src/utils/pathUtils.ts

```diff
@@ -2,6 +2,7 @@ import toPath from 'lodash/toPath';
 import get from 'lodash/get';
 import set from 'lodash/set';
 import invariant from 'tiny-invariant';
+import { ROOT_PATH } from '../hooks';
```

Comment:
Please, avoid importing code from `hooks/index.ts`, `components/index.ts` or `typings/index.ts` as this may lead to circular dependencies.